doc/install.rst

```diff
@@ -94,35 +94,63 @@ Currently, ``xgboost-cpu`` package is provided for x86_64 (amd64) Linux and Wind
 Conda
 *****
 
-You may use the Conda packaging manager to install XGBoost:
+The `Conda <https://conda.io>`_ packaging manager provides a language agnostic, cross-platform friendly, ``sudo``-free install experience.
+
+If you don't already have Conda, it can be installed via `the conda-forge installer, Miniforge <https://conda-forge.org/download/>`_. 
+
+To install Python XGBoost:
 
 .. code-block:: bash
 
-   conda install -c conda-forge py-xgboost
+   conda install py-xgboost
```

Comment:
Do we need to clarify what the package `xgboost` (without the language prefix) refers to? https://anaconda.org/conda-forge/xgboost